doc/source/reference/ufuncs.rst

```diff
@@ -327,6 +327,12 @@ advanced usage and will not typically be used.
     multiple outputs is deprecated, and will raise a warning in numpy 1.10,
     and an error in a future release.
 
+    If 'out' is None (the default), a uninitialized return array is created.
+    The output array is then filled with the results of the ufunc in the
+    places that 'where' is True, by default everywhere if 'where' is not
```

Comment:
Possibly "that the broadcasted 'where' is True, which by default is everywhere."?

Reply:
So perhaps:

> The output array is then filled with the results of the ufunc in the places that the broadcast 'where' is True. If 'where' is True (the default), then this corresponds to the entire output being filled.

Reply:
adopted